powermon/config/config_model.py

```diff
@@ -30,7 +30,7 @@ class APIConfig(NoExtraBaseModel):
     """ model/allowed elements for api section of config """
     host: None | str
     port: None | int
-    enabled: None | bool
+    enabled: None | bool = Field(default=False)
```

Comment:
what does this do?

Reply:
If the config isn't provided it sets the default to False. I suddenly found my config failing due to missing values. I figured sensible defaults could be added.